src/main/resources/anniv5Resources/localization/eng/serpentinepack/UIstrings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "${ModID}:SerpentinePack": {
+    "TEXT": [
+      "Serpentine Disciple",
+      "A disciple of the reptomancer. Uses Venemous and Cunning stances which focus on applying debuffs.",
+      "Dayvig",
+      "Pack Art - Dayvig"
+    ]
+  },
+  "${ModID}:SerpentinePackHat": {
+    "TEXT": [
+      "Like the Watcher's, but with noticeably less red.",
+      "Sentinel Sash"
```

Comment:
I don't see the hat anywhere?

Reply:
This is also a copy pasta from the Sentinel pack's hat text, so it looks like all hat stuff is MIA.  Which can be fine - AlexMdle will add a hat if you ask.